elliottlib/bzutil.py

```diff
@@ -1063,8 +1063,17 @@ def is_first_fix_any(bugtracker, flaw_bug, current_target_release):
         return True
 
     # filter tracker bugs by OCP product
-    tracker_bugs = [b for b in bugtracker.get_bugs(tracker_ids)
-                    if b.product == constants.BUGZILLA_PRODUCT_OCP and b.is_tracker_bug()]
+    tracker_bugs = []
+    for tracker_id in tracker_ids:
+        try:
+            b = bugtracker.get_bug(tracker_id)
+        except Exception as e:
```

Comment:
Is there a less generic Exception we can catch?

Reply:
there is https://github.com/openshift/elliott/blob/master/elliottlib/exceptions.py#L31 but seems just a mask